src/metrics.go

```diff
@@ -119,7 +128,7 @@ func getPlusMetrics(reader *bufio.Reader) (map[string]interface{}, error) {
 		return nil, err
 	}
 
-	roots := [2]string{"connections", "requests"}
+	roots := [4]string{"connections", "requests", "ssl", "processes"}
```

Comment:
could we remove the 4? It seems redundant

Reply:
yeah. I just left the size because it was there before.